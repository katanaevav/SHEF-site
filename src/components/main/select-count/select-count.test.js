import React from "react";
import renderer from "react-test-renderer";
import SelectCount from "./select-count.jsx";

it(`Render Select Count`, () => {
  const tree = renderer.create(
    <SelectCount
      defaultValue = {1}
      minValue = {1}
      maxValue = {99}
      onChangeValue = {()=>{}}
    />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
