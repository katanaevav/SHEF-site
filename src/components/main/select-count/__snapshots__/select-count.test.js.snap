// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Select Count 1`] = `
<div
  className="dish__counter select-count"
>
  <button
    className="select-count__button select-count__button--dec"
    onClick={[Function]}
  >
    -
  </button>
  <input
    className="select-count__input"
    defaultValue={1}
    max={99}
    min={1}
    onChange={[Function]}
    type="number"
  />
  <button
    className="select-count__button select-count__button--inc"
    onClick={[Function]}
  >
    +
  </button>
</div>
`;
